### DISCUSSION TITLE
How off private on territory for enemy

### DISCUSSION THREAD

**Discussion by mopel63** [TownyAdvanced/Towny#7635] [Q&A]
When I try to view the settings, I see everything, but I don't see the enemy's private rights settings.
![Screenshot_10](https://github.com/user-attachments/assets/fbb7fd5c-1011-4d18-9551-e449c27e6f82)

**Comment by LlmDl** [comment]
Enemies usually fall under outsiders.

**Reply by mopel63** [reply]
It’s good if you change this setting, then when the player has not yet become an enemy, he can still break blocks in any citying block

**Comment by LlmDl** [comment]
Players have a cache which stores their ability to do things in plots.
Usually that cache would get recreated when something changes. I will check
if we do that when players are made into enemies.

**Reply by mopel63** [reply]
Ok thanks

**Reply by mopel63** [reply]
You check it?

**Reply by LlmDl** [reply]
I did check it, when a nation adds or removes an enemy the entire plugin's player cache is reset.

If you have an issue file an issue ticket and don't skip the form. Make sure you leave precise instructions on how to reproduce the issue.
